docker-compose/Makefile

```diff
@@ -39,10 +39,10 @@ quick-start-info:
 	echo "$$(tput setaf 2)To invoke the function again use: $$(tput setaf 4)make hello-world$$(tput sgr0)"
 	echo "$$(tput setaf 2)To stop openwhisk use: $$(tput setaf 4)make destroy$$(tput sgr0)"
 
-docker:
-	echo "building the docker images ... "
+init-cli:
```

Comment:
Should we preserve `make docker` for local development use-case and add `init-cli` in addition ?